### ISSUE TITLE
Allowing a "multiplier factor" to POLYMOD data based contact model

### ISSUE THREAD

**Issue by mikkokotila** [kausaltech/reina-model#25]
Related with #24

One thought is to allow applying a "multiplier" to the POLYMOD data. Assume that relatively speaking it's meaningful, but then allow the person who runs the simulation to apply a factor to it that increases it. For example, if you want to consider things like standing in the public transport face to face with someone for a period of time, and whatever else it might be, then just apply the factor as you like. That way we stay out of it but acknowledge that it might be relevant to consider a broader definition of a "contact" than the one that is used in POLYMOD. I think it's important to keep the goal in mind here; to influence decisions. If decisions are influenced, and they are influenced based on data that poorly reflects what is actually relevant (e.g. it's probably more relevant that you stay 10 minutes in direct gas exchange with someone, than the fact that you exchange a few words with them).

**Comment by jampekka** [comment]
Yes, a multiplier for "contagiousness" of a contact would be good. There's also contact duration estimates in he POLYMOD that could be used (longer duration more likely leads to a transmission).

More "casual" contacts, like standing near somebody without actually contacting them verbally or physically, will not be included in the POLYMOD due to its design. It could be compensated for by e.g. some multipliers, but ideally there would be an another dataset for "general proximity" of people.

A big problem in this is that we don't really know how contagious different kinds of contacts are (making estimating the multipliers difficult), and more generally how much the virus spreads by "indirect contacts" such as touching same surfaces. The current implicit assumption is that all of these are correlated enough to the reported contacts to be usable in the simulation.
